lib/rubocop/cop/rails/pluck.rb

```diff
@@ -9,6 +9,9 @@ module Rails
       # element in an enumerable. When called on an Active Record relation, it
       # results in a more efficient query that only selects the necessary key.
       #
+      # @safety
+      # This cop's autocorrection is unsafe because model can define column aliases.
```

Comment:
Can you tweak and indent the text? Perhaps example code would be useful:

```ruby
# @safety
#   This cop is unsafe because ...
#
#   [source,ruby]
#   ----
#   'please replace with an example code for unsafe'
#   ----
```